src/components/Layout/index.js

```diff
@@ -12,7 +12,7 @@ class Layout extends React.Component {
   componentWillMount () {
     WebFont.load({
       google: {
-        families: ['El Messiri:300,400,700', 'Raleway:300,400,400i,700,700i']
+        families: ['El Messiri:300,400,700', 'Raleway:300,400,400i,600,700,700i']
```

Comment:
We can also think about hadding intelligent lodaing on font here. If language is arabic load El Messiri etc.

Reply:
Jap. I would try to do this with WEBAPP-96